.autorc

```diff
@@ -0,0 +1,16 @@
+{
+    "onlyPublishWithReleaseLabel": true,
+    "baseBranch": "master",
+    "author": "auto <auto@nil>",
+    "noVersionPrefix": false,
+    "plugins": [
+        "git-tag",
+        [
+            "exec",
+            {
+                "afterChangelog": "bump2version \"$(printf '%s\n' \"$ARG_0\" | jq -r .bump)\"",
```

Comment:
could you please remind me why we need the bump2version?

Reply:
ah, because of no versioneer or alike used here, right?

Reply:
Correct.  pyout sets its version directly in `setup.py` and `pyout/__init__.py`, so we use bump2version to increment it.